src/store/reducers/medCenterReducer.ts

```diff
@@ -0,0 +1,37 @@
+const ADD_MEDCENTER = 'ADD_MEDCENTER';
+
+interface IMedCenterData {
+  medCenterData: any[];
+}
+
+interface IAction {
+  type: string;
+  payload?: any;
+}
```

Comment:
same as above, no `any`'s, type should be a union with the set of possible values